.github/workflows/rust-tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Rust Tests
```

Comment:
Is it worth expanding the name to detail that they are integration tests?

Reply:
The workflow just runs `cargo test`, which includes all the tests written in Rust.